mkosi/__init__.py

```diff
@@ -4126,28 +4127,27 @@ def secure_boot_sign(
         return
 
     with mount():
-        for path, _, filenames in os.walk(root / "efi"):
-            for i in filenames:
-                if not i.endswith(".efi") and not i.endswith(".EFI"):
-                    continue
+        for f in itertools.chain(efi_path.glob('**/*.efi'), efi_path.glob('**/*.EFI')):
+            if os.path.exists(f"{f}.signed"):
```

Comment:
```suggestion
            if f.with_suffix(".efi.signed").exists()
```

Reply:
If you insist I'll change it, but this needs to be `if f.with_suffix(".efi.signed").exists() or if f.with_suffix(".EFI.signed").exists()`. I'm not seeing the benefit beyond casting a path to a string.

Reply:
Nevermind then, I forgot about the blasted case insensitivity of FAT32 again. `Path`s really need an `add_suffix` method.